analyzers/tests/SonarAnalyzer.UnitTest/TestCases/LinkedListPropertiesInsteadOfMethods.vb

```diff
@@ -0,0 +1,25 @@
+﻿Imports System.Collections
```

Comment:
Yes they are not in sync because they are a bit different. In CSharp `data.First()` is actually accessing the Enumerable.First, the same line in VB.NET should be compliant because it's actually calling the First property of the LinkedList (see [this comment](https://github.com/SonarSource/sonar-dotnet/pull/7180#discussion_r1190269367) for more)